test/configuration/configuration.test.ts

```diff
@@ -84,4 +85,10 @@ suite('Configuration', () => {
     end: ['|'],
     endMode: ModeName.Visual,
   });
+  
+  test('Can handle custom digraph insert', async () => {
```

Comment:
With the latest release, this tests fails since nothing gets inserted in the buffer. Do you know what might cause this?

Reply:
You can take a look at the changelog, but nothing apparent comes to mind.